packages/config/index.js

```diff
@@ -28,6 +28,13 @@ const config = convict({
     env: 'NODE_ENV'
   },
 
+  flavor: {
```

Comment:
As I said in readme, I think calling this flavor makes it hard to understand that it;'s about the source of where comments come from... I suggest naming it more explicitly and/or adding a comment to explain what it is.

Reply:
See above: I think I'll get rid of this setting entirely.  I now think it is a bad idea.